Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_light_rifle.yml

```
@@ -173,3 +173,184 @@
     - state: mag-1
       map: ["enum.GunVisualLayers.Mag"]
     - state: uranium
+
+#Magazine
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)